### ISSUE TITLE
bem-font-awesome-icons: второй подход к снаряду

### ISSUE THREAD

**Issue by tadatuta** [bem-site/bem-forum-content-ru#1274]
Буквально на днях мы [анонсировали](https://ru.bem.info/forum/1272) библиотечку bem-font-awersome, которая предоставляет возможность использовать [Font Awesome](http://fontawesome.io/) с использованием БЭМ-нотации и без необходимости тянуть лишние стили.

На этот раз мы пошли дальше и распилили шрифт на отдельные SVG-иконки, так что теперь на клиент приедет только то, что реально используется.

Новую библиотеку назвали bem-font-awesome-icons.
Она предоставляет иконки в виде модификаторов блока `icon` в двух вариантах: как фоновая картинка (модификатор `bg`) и инлайном через шаблоны BEMHTML и BH, чтобы иконки можно было стилизовать через CSS (через модификатор `glyph`).

Поставляется библиотека через `npm` или `bower`.

Подробности см. в документации: https://github.com/tadatuta/bem-font-awesome-icons

**Comment by DjonyBastone** [comment]
Круто!! спасибо ребята.
В самый подходящий момент.

Недавно подключал сервис яндекса - поделиться ссылкой. Там код по методологии БЭМ. Обнаружил недостаток передачи иконки как background: url(...).  При такой передаче svg можно поменять только - фон, на саму иконку (fill)  повлиять не возможно.
Было бы круто оставить возможность воздействия на все ингредиенты иконки. Если есть какие-нибудь мысли по этому поводу - пожалуйста поделитесь

**Comment by Realetive** [comment]
@DjonyBastone https://github.com/bem-site/bem-forum-content-ru/issues/1247

**Comment by DjonyBastone** [comment]
@Realetive Да, именно так - как и думал. Только в спрайты не заключил свои кастомные иконки, использую инлайн SVG для управления всеми стилями. Думал к поднакопить иконки и перед продакшн сделать спрайт.

Предыдущее сообщение в большей степени адресовано ребятам из Яндекс.Сервисов. Пожелание к сервисам - оставьте возможность получать svg инлайн !! пойду продублирую в блок "поделиться"

**Comment by rteamx** [comment]
Thanks, it's fucking awesome! :)
